designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/theme/TextColor.kt

```diff
@@ -2,7 +2,7 @@ package org.hisp.dhis.mobile.ui.designsystem.theme
 
 object TextColor {
     val OnPrimary = Color.NeutralWhite
-    val OnPrimaryContainer = Color.Blue900
+    val OnPrimaryContainer = Color.Blue700
```

Comment:
Is this validated with Marcos?

Reply:
I don't know if it was modified from when it was first done but according to figma that is the blue that is required for that color tag